src/components/elements/NftSearchBox.stories.tsx

```diff
@@ -1,11 +1,13 @@
-import React from "react";
 import { storiesOf } from "@storybook/react";
+import { IdentityThemeProvider } from "../../themes";
```

Comment:
I'm thinking of adding a global storybook decorator that wraps every story in `IdentityThemeProvider`, can't think of a reason not to, what do you think?

Reply:
Yes! That's a great idea. I can't think of any reason we wouldn't want it globally.